pkg/deploy/controller/generictrigger/controller.go

```diff
@@ -213,6 +213,7 @@ func (c *DeploymentTriggerController) update(config *deployapi.DeploymentConfig,
 		config.Status.Details.Message = "caused by an image change"
 	}
 	_, err := c.dn.DeploymentConfigs(config.Namespace).UpdateStatus(config)
+	glog.V(5).Infof("Failed to update deployment config %s/%s status %+#v: %v", config.Namespace, config.Name, config.Status, err)
```

Comment:
This is not correct as we are returning the error unconditionally. You may want to log in its caller.